learn/what_is_meilisearch/features.md

```diff
@@ -32,15 +32,15 @@ Read more about synonyms in [this dedicated guide](/reference/features/synonyms.
 
 ## Filters
 
-Meilisearch allows you to define [filters](/reference/features/filtering.md) so you can filter through the results based on criteria.
+Meilisearch allows you to define [filters](/reference/features/filtering_and_faceted_search.md) so you can filter through the results based on criteria.
```

Comment:
Yeah, criteria seems a bit vague and just hanging off the end of this sentence. 

I'm not sure I like `sorting condition` because it introduces a concept that is not really expanded upon in the rest of the page. How do you feel about `user-defined criteria`?

```suggestion
MeiliSearch allows you to define [filters](/reference/features/filtering_and_faceted_search.md) so you can filter through the results based on user-defined criteria.
```